docs/deploy/baseten.md

```diff
@@ -37,3 +37,22 @@ baseten.deploy_truss(my_truss)
 ```
 
 Head over to [your Baseten account](https://app.baseten.co) to see the model deployment logs and interface with your newly deployed model!
+
+### Deploying with secrets
+
+If your model uses [secrets](../develop/secrets.md), set `trusted=True` in the `deploy_truss` command to enable your model to access secrets:
+
+```python
+import baseten
+basten.deploy_truss(
+    my_truss,
+    model_name="My Model",
+    trusted=True
+)
+```
+
+Secrets can be securely stored in your Baseten organization by following [this documentation](https://docs.baseten.co/applications/files/secret-management).
+
+{% hint style="warning" %}
+Baseten mounts secrets, so do not use the `TRUSS_SECRET_` prefix when setting secret names.
```

Comment:
May be worth clarifying why users would understand the `TRUSS_SECRET_` reference in the context of this documentation, sth that points out `unlike when truss secrets are bound via environment variables`.